R/utils.R

```diff
@@ -407,4 +407,58 @@ findColNamesByPredicate <- function(variableList, predicateFunction) {
 
 avgDigits <- function(x) {
   floor(mean(stringi::stri_count_regex(as.character(x), "[[:digit:]]")))
+}
+
+
+#
+# For any number, return an absolute delta (numeric) at the last
+# significant digit in the number, using the number of digits specified
+#
+# e.g. assuming 3 significant digits
+# 
```

Comment:
We should add tests in test-utils for all of these example.

Reply:
Good idea will do.

Is this more of an Epsilon than Delta?

Reply:
oh yes you're right!

Reply:
Thanks again for your review and approval (pending all the suggestions you made)
I will aim to get this rolled out to the dfalke-b back end on Monday where I can check more easily that the bins look OK.

Reply:
Not moving it at this point (see https://github.com/VEuPathDB/plot.data/issues/188) but renamed and added tests :heavy_check_mark: